README.md

```diff
@@ -28,27 +28,22 @@ After that, follow the next steps:
 
 - If you're receiving a `[SequelizeConnectionError]: password authentication failed for user '<username>'`, please enter to postgres console using `sudo -u postgres psql` and change the password for the user you're trying to use. Also update .env accordingly
 
-# Manual deployment
+# Automatic Deployment
 
-In order to deploy the project to Google Cloud App Engine, you'll need a project with the following pre-configured APIs and resources:
+In order to deploy the project to Google Cloud App Engine, you'll need a GCloud project with the following pre-configured APIs and resources:
 
 - A Cloud SQL instance running PostgreSQL
 - An App Engine setup for a Node JS service
+- Big Query enabled in your GCloud Project
 
-Open your datastore in Google Cloud, create a collection named `secrets`, then create an entry with the database variables according to [the official docs](https://cloud.google.com/sql/docs/postgres/connect-app-engine-standard?hl=es-419#node.js).
+We use GCloud Secret Manager to store secret keys needed, so, you'll need to add every environmental variable specified in `.env.example` inside the Secret Manager of your GCloud project except for `NODE_ENV` and `PORT` and client side env variables. These variables are set by App Engine by default or directly not needed.
```

Comment:
I'm not sure I'm parsing the last sentence here (These variables...). Is the word "directly" needed?

Reply:
I made a bad spanish-english translation haha
Done!